src/server.js

```diff
@@ -360,6 +368,35 @@ function encodeAddress(address) {
   return formattedAddress;
 }
 
+app.get(`${PREFIX_API}/reverseGeocode`, async (request, response) => {
```

Comment:
Same comment as on Asha's PR. I think it makes more sense to do the reverse geo-coding on the server as part of the participants response itself, since it can be parallelized easier, could be cached, etc.

Basically I'm saying the client shouldn't call a reverseGeocode api on the server. Instead, the server should just make the requests when returning the participants data.

Reply:
I feel it makes sense to do it here because it may be more complicated on the server. I am only trying to reverse-geocode  when the user doesn't enter a physical address. What are the benefits of having it parallelized or cached?

Reply:
I don't think it would be more complicated to do on the server. The benefit is as follows: Instead of doing the reverse geocoding everytime someone visits the participants page, we can just do it once when a participant updates their long/lat. This saves API calls & improves performance.

Reply:
The reverse geocoding API is not called every time someone visits the participant page. Its only called when the user uses the HTML geolocation to update their long lat, as you mentioned.

Reply:
Or maybe I'm not getting the point. We can talk more about this over hangout

Reply:
No, you're correct. I misunderstood!

Reply:
Great!

Reply:
Are there other changes I need to make?